core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -332,23 +332,23 @@ class IndexingQueueActor(searchService: SearchService) extends Actor with ActorL
           else searchService.extractSymbolsFromClassOrJar(f).map(f -> )
       }).onComplete {
         case Failure(t) =>
-          log.error(s"failed to index batch of ${batch.size} files", t)
+          log.error(t, s"failed to index batch of ${batch.size} files")
         case Success(indexed) =>
           searchService.delete(indexed.map(_._1)(collection.breakOut)).onComplete {
             case Failure(t) =>
               searchService.semaphore.release()
-              log.error(s"failed to remove stale entries in ${batch.size} files", t)
+              log.error(t, s"failed to remove stale entries in ${batch.size} files")
             case Success(_) => indexed.foreach {
               case (file, syms) =>
-                val boost = searchService.isUserFile((file.getName))
```

Comment:
this double parenthesis is needed for the presentation compiler to understand it